packages/rune-games-cli/src/lib/validateGameFiles.ts

```diff
@@ -112,7 +108,14 @@ export async function validateGameFiles(
         .getAttribute("src")
         ?.match(sdkVersionRegex)?.[1]
 
-      const sdkVersionCoerced = semver.coerce(sdkVersion)
+      if (!sdkVersion) {
+        errors.push({ message: `Rune SDK must specify a version` })
+      }
+
+      const [major, minor, patch] = (sdkVersion ?? "")?.split(".")
+      const maxedOutSdkVersion = `${major}.${minor ?? 999}.${patch ?? 999}`
```

Comment:
why do we do 999?

Reply:
to allow comparing e.g. `sdk@4` with `minVersion: 4.5.0`. semver package resolves `4` to `4.0.0` but it's actually "the latest 4.x.x" version

Reply:
I see, missed the variable name, thanks :)